src/qt/transactionview.cpp

```diff
@@ -742,3 +746,16 @@ void TransactionView::updateWatchOnlyColumn(bool fHaveWatchOnly)
     watchOnlyWidget->setVisible(fHaveWatchOnly);
     transactionView->setColumnHidden(TransactionTableModel::Watchonly, !fHaveWatchOnly);
 }
+
+void TransactionView::updatePrivateSendVisibility()
+{
+    bool fEnabled = CPrivateSendClientOptions::IsEnabled();
+    // If PrivateSend gets enabled use "Most common" else "All"
+    typeWidget->setCurrentIndex(fEnabled ? 1 : 0);
```

Comment:
It feels like this should be the other way around tbh
```suggestion
    // If PrivateSend gets enabled use "All" else "Most common"
    typeWidget->setCurrentIndex(fEnabled ? 0 : 1);
```

Reply:
oh right, sounds better 😄